### PR TITLE
converted TIDashboard link to button and made NEXT as SAVE AND NEXT

### LINKED ISSUE #1742: Improve TI Dashboard Button
The process for accessing the "TI Dashboard" isn't intuitive. Is it possible to add a more obvious button for "TI Dashboard" at the top of the screen after TIs login? There is confusion about where to go and the text at the top of the screen isn't obvious that it's a button.

### LINKED ISSUE #2098: Unclear that "Next" Is really "Save & Next"
It needs to be clear that in order for CiviForm to save a resident's answers, they need to select "Next". Is there a way to change the wording on the button so that it's clear that "next" is really a "save & next"? We have residents going back to previous pages and being confused why answers are deleted.

**Describe the solution you'd like**
Change button text to "Save & Next" or something like that!
Save and Next

**Additional context**

**Done when**
What conditions need to be met before this issue can be closed?

### PR DESCRIPTION
**Pull request by swatkat1**
### Description
This PR has two changes:
1. The NEXT button the user sees while filling the applications is not clear if the application till the point was saved or not. So the change is to make the NEXT button as "SAVE AND NEXT".
2. The second change is is the TI home page. The TI Dahsboard link is not clearly highlighted as a button, so I have modified it to be like a SUBMIT button style. 

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary
Screen shot:
image

image

Please note - I am unable to test the TI changes locally. 
### Issue(s)

Fixes #1742 ; Fixes #2098

### CODE REVIEW

**Comment by shanemc-goog** [comment]
Looks like the browser tests are failing.

File: universal-application-tool-0.0.1/conf/messages.en-US [hunk 1]
```diff
@@ -12,7 +12,7 @@ button.continue=Continue
 button.deleteFile=Delete
 button.keepFile=Continue
 button.logout=Logout
-button.nextScreen=Next
+button.nextScreen=Save And Next
```
Comment (goldblatt) [inline universal-application-tool-0.0.1/conf/messages.en-US:15]: probably would want the 'And' to be lowercased?

**Review by goldblatt — CHANGES_REQUESTED** [review]
could you include a screen shot with the new changes!

File: universal-application-tool-0.0.1/conf/messages.en-US (hunk 1 above)
Comment (swatkat1) [reply universal-application-tool-0.0.1/conf/messages.en-US:15]: Everything is shown as upper case text in UI

Comment (swatkat1) [reply universal-application-tool-0.0.1/conf/messages.en-US:15]: Everything is shown as upper case text in UI

File: browser-test/src/support/applicant_questions.ts [hunk 2]
```diff
@@ -191,7 +191,7 @@ export class ApplicantQuestions {
   }
 
   async clickNext() {
-    await this.page.click('text="Next"')
+    await this.page.click('text="Save And Next"')
```
Comment (shanemc-goog) [inline browser-test/src/support/applicant_questions.ts:194]: Are you working with Elise on translations for this?

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java [hunk 3]
```diff
@@ -178,12 +176,13 @@ private ContainerTag maybeRenderTiButton(Optional<CiviFormProfile> profile, Stri
...
           a(tiDashboardText)
-              .withHref(tiDashboardLink)
-              .withClasses(
-                  Styles.PX_3,
-                  Styles.TEXT_SM,
-                  Styles.OPACITY_75,
-                  StyleUtils.hover(Styles.OPACITY_100)),
+              .with(
+                  new LinkElement()
+                      .setId("ti-Dashboard")
```
Comment (shanemc-goog) [inline universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:181]: lowercase dashboard, ids are all lowercase

**Review by shanemc-goog — CHANGES_REQUESTED** [review]

File: browser-test/src/support/applicant_questions.ts (hunk 2 above)
Comment (swatkat1) [reply browser-test/src/support/applicant_questions.ts:194]: Yes.

Comment (swatkat1) [reply browser-test/src/support/applicant_questions.ts:194]: Yes.

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java (hunk 3 above)
Comment (swatkat1) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java]: done

**Review by shanemc-goog — APPROVED** [review]

**Pull request merged**